views/index.ejs

```diff
@@ -8,12 +8,12 @@
     <link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
   </head>
   <body>
-    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
-    <script src="https://code.jquery.com/ui/1.12.1/jquery-ui.min.js"></script>
-    <script src="https://unpkg.com/typewriter-effect/dist/core.js"></script>
-    <script src="scripts/confetti.js"></script>
-    <script src="scripts/history.js"></script>
-    <script src="scripts/main.js"></script>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js" defer></script>
+    <script src="https://code.jquery.com/ui/1.12.1/jquery-ui.min.js" defer></script>
+    <script src="https://unpkg.com/typewriter-effect/dist/core.js" defer></script>
+    <script src="scripts/confetti.js" defer></script>
+    <script src="scripts/history.js" defer></script>
+    <script src="scripts/main.js" defer></script>
```

Comment:
Changes look good. :+1: 
If possible edit your description to use one of the special keywords, it will automatically link the issue and make it easier for you to get credit for the PR.